packages/catalog-client/api-report.md

```diff
@@ -145,14 +153,30 @@ export interface CatalogRequestOptions {
 export const ENTITY_STATUS_CATALOG_PROCESSING_TYPE =
   'backstage.io/catalog-processing';
 
+// @public
+export type EntityFieldsQuery = string[];
+
+// @public
+export type EntityFilterQuery =
+  | Record<string, string | symbol | (string | symbol)[]>[]
+  | Record<string, string | symbol | (string | symbol)[]>;
+
+// @public
+export interface GetEntitiesByRefsRequest {
+  entityRefs: string[];
+  fields?: EntityFieldsQuery | undefined;
+}
+
+// @public
+export interface GetEntitiesByRefsResponse {
+  items: Array<Entity | undefined>;
```

Comment:
I noticed this was `Entity | null` in the other PR does it matter?

Reply:
It's just that null is the only one that is json serializable - it doesn't matter much but I thought undefined was the most "javascriptish" and in the spirit of avoiding null in general (which seems to be a common code health idea).

Reply:
(we return undefined for 404s in all the other methods too, so it's consistent with the rest of the interface too)